content/guides/tasks/assignments/index.md

```diff
@@ -0,0 +1,14 @@
+---
+rank: 1
+related_endpoints: []
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Assigning tasks
+
+When a task is delegated to a user in Box, that creates a task assignment.
+This can be used to denote a review action a user needs to take, such as
+providing input on a legal contract. A task assignment can also be given a date
+it should be completed by.
```

Comment:
So... is the type of action a task expects related to the task or the task assignmnent? Do we want to expand on this and send people to the task index page to learn more about different types of tasks?

Reply:
Right now, we only have "review" task types. That type of task (being tied to a specific user) is a task assignment, though the date is tied to the task id, not the task assignment id

Reply:
Ok. I guess I'm having trouble understanding the different resolution_states mentioned here.

Reply:
Made some small language changes and we can include resolution states on the index page if you want, but for now I've put this context in the guide and in the OAS updates.